zfit/core/limits.py

```diff
@@ -0,0 +1,476 @@
+from __future__ import print_function, division, absolute_import
```

Comment:
This is one of the modules where a general module-level doc, explaining a bit the Range object, would be very nice (although not necessary at this stage).

Reply:
Done